### ISSUE TITLE
Automatically switch between dark and light mode

### ISSUE THREAD

**Issue by kvhjdffv** [wotakumoe/wotaku#76]
Hello, I was wondering if you could implement something that will:
1. Automatically detect whichever mode a user is using
2. Automatically switch to that mode

Examples:
![2024-10-29 14_47_08-Window](https://github.com/user-attachments/assets/8b25efd7-b982-4a2d-b04b-9c06beedb5b0)
*(Light mode)*

Or
![2024-10-29 14_48_05-Window](https://github.com/user-attachments/assets/b9f935dc-5eb6-421e-bbdb-d9b99c8c43da)
*(Dark mode)*

In other words, could you please set the default mode to Automatic instead of Dark? Thanks.
